lib/rohd_hcl.dart

```diff
@@ -1,10 +1,10 @@
 // Copyright (C) 2023-2024 Intel Corporation
 // SPDX-License-Identifier: BSD-3-Clause
 
-export 'src/adder.dart';
 export 'src/arbiters/arbiters.dart';
+export 'src/arithmetic/adder.dart';
```

Comment:
Why is adder.dart not part of the arithmetic.dart exports?

Reply:
I missed this